apps/spruce/src/pages/projectSettings/tabs/GithubAppSettingsTab/getFormSchema.tsx

```diff
@@ -184,7 +210,7 @@ const appFieldCss = css`
   max-width: unset;
 `;
 
-const StyledDescription = styled.span`
+const StyledDescription = styled.div`
```

Comment:
this has to be a span to avoid nesting errors (it's invalid HTML, error can be seen in the console)
```suggestion
const StyledDescription = styled.span`
```